packages/opentelemetry-node/README.md

```diff
@@ -57,7 +57,7 @@ const opentelemetry = require('@opentelemetry/core');
 const { NodeTracer } = require('@opentelemetry/node');
 
 // Create and configure NodeTracer
-const tracer = new NodeTracer({
+const tracerFactory = new NodeTracerFactory({
```

Comment:
Do we suggest to users to do `new NodeTracerFactory(options)` or `NodeTracerFactory.instance(options)` or `getTracerFactory(options)`?